pkg/eden/qemu.go

```diff
@@ -41,10 +42,39 @@ func StopSWTPM(stateDir string) error {
 	return utils.StopCommandWithPid(pidFile)
 }
 
+func startQMPLogger(qmpSock string, qmpLog string) error {
+	shellcmd := fmt.Sprintf(
+		"echo '{\"execute\": \"qmp_capabilities\"}' | " +
+		"socat -t0 -,ignoreeof UNIX-CONNECT:%s > %s",
+		qmpSock, qmpLog)
+	opts := []string{
+		"-c", shellcmd,
+	}
+
+	var err error
+
+	// Retry a few times if socket is not available yet
+	n := 5
```

Comment:
I suspect that on these runners we may need to give more than 5 seconds for qemu to start.